tests/test_loader.py

```diff
@@ -90,13 +107,13 @@ def test():
     hv = sha1sum.hexdigest()
 
     with pytest.raises(ImportError):
-        import inmanta_unit_test  # NOQA
+        import inmanta_plugins.inmanta_unit_test  # NOQA
 
-    cl.deploy_version(hv, "inmanta_unit_test", code)
+    cl.deploy_version(hv, "inmanta_plugins.inmanta_unit_test", code)
 
-    import inmanta_unit_test  # NOQA
+    import inmanta_plugins.inmanta_unit_test  # NOQA
 
-    assert inmanta_unit_test.test() == 10
+    assert inmanta_plugins.inmanta_unit_test.test() == 10
 
     # reload cached code
     cl.load_modules()
```

Comment:
could you also test updating the content of the module and loading it again, to verify correct re-loading?